content/collections/session-replay/en/session-replay-ios-plugin.md

```diff
@@ -81,11 +98,63 @@ let amplitude = Amplitude(configuration: Configuration(apiKey: API_KEY))
 amplitude.add(plugin: AmplitudeSwiftSessionReplayPlugin(sampleRate: 1.0))
 ```
 
+{{/partial:tab}}
+{{partial:tab name="Middleware"}}
```

Comment:
Instead of calling this middleware and plugin, can we use the analytics SDK names? Swift SDK vs legacy SDK? I dont think we call out when we use a middleware vs a plugin anywhere. So this can be a little confusing.